aptos-move/aptos-vm/src/data_cache.rs

```diff
@@ -113,17 +122,30 @@ impl<'a, S: StateView> StorageAdapter<'a, S> {
                 return Ok((buf, buf_size));
             }
             let group_data = self.get_resource_group_data(address, &resource_group)?;
-            if let Some(group_data) = group_data {
+            if let Some(group_data_blob) = group_data {
+                let common_error = || {
```

Comment:
Unless it is not backwards compatible...

Reply:
@runtian-zhou @zekun000 @msmouse : would adding a message here be backwards compatible?

Reply:
Reading the code I think it won't (because [the abort info doesn't go into on-chain status](https://github.com/aptos-labs/aptos-core/pull/9800/files/a03bafa43b362bca1df5ce5a45a31d600c5bbced)) but @runtian-zhou  is the expert.

Reply:
Well I thought you were the expert Alden! Unfortunately I think the abort info is indeed part of the txn info that gets persisted into the ledger. Code: https://github.com/aptos-labs/aptos-core/blob/c2153785ea0bacf37a579971e9f38fc26aaeaf2e/types/src/transaction/mod.rs#L843

Can you elaborate when this error could occur? If it is just BCS error I don't think they could happen?

Reply:
Yea I mean we deliberately left it to be always None?

@wrwg @vgao1996   I'm under the impression that we thought `AbortInfo` being part of `TxnInfo` was a mistake?

Reply:
The abort_info does not need to be persisted IIRC. But it also must not be lost as it travels through the API layers to developers. Do not have enough context to say what to do in this particular use case.